website/docs/d/virtual_desktop_host_pool.html.markdown

```diff
@@ -0,0 +1,87 @@
+---
+subcategory: "Desktop Virtualization"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_virtual_desktop_host_pool"
+description: |-
+  Gets information about an existing Virtual Desktop Host Pool.
+---
+
+# Data Source: azurerm_virtual_desktop_host_pool
+
+Use this data source to access information about an existing Virtual Desktop Host Pool.
+
+## Example Usage
+
+```hcl
+
+data "azurerm_virtual_desktop_host_pool" "example" {
+  name                = "example-pool"
+  resource_group_name = "example-resources"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Virtual Desktop Host Pool. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which to create the Virtual Desktop Host Pool. Changing this forces a new resource to be created.
```

Comment:
the Data Source looks up information about an existing Resource, so the tone wants to be slightly different here:

```suggestion
* `name` - (Required) The name of the Virtual Desktop Host Pool to retrieve.

* `resource_group_name` - (Required) The name of the resource group where the Virtual Desktop Host Pool exists.
```